src/Components/Input.tsx

```diff
@@ -12,276 +10,57 @@ export interface InputProps extends React.HTMLProps<HTMLInputElement> {
-            <InputComponent
-              innerRef={this.props.innerRef}
-              {...newProps}
-              onFocus={this.onFocus}
-              onBlur={this.onBlur}
-              onChange={this.onChange}
-              value={this.state.value}
-              type={this.convertedType}
-              showLabel={showLabel}
-            />
-            {isPassword
-              ? this.getRightViewForPassword()
-              : !!rightView && rightView}
-          </InputContainer>
-          {!error && showPasswordMessage ? (
-            <PasswordMessage>
-              Password must be at least 8 characters.
-            </PasswordMessage>
-          ) : (
-            ""
-          )}
-          <Error show={!!error}>{error}</Error>
-        </Container>
-      )
-    }
-
-    const { title, description, ref, ...rest } = this.props
-    return (
-      <Container>
-        {title && <Title>{title}</Title>}
-        {description && <Description>{description}</Description>}
-        <StyledInput hasError={!!error} {...rest} />
-        <Error show={!!error}>{error}</Error>
-      </Container>
-    )
-  }
+export const Input: SFC<InputProps> = ({
+  error,
+  title,
+  description,
+  ref,
+  ...rest
+}) => {
+  return (
+    <Container>
+      {title && <Title>{title}</Title>}
+      {description && <Description>{description}</Description>}
+      <StyledInput hasError={!!error} {...rest} />
+      {error && <InputError>{error}</InputError>}
+    </Container>
+  )
 }
 
 const Container = styled.div`
-  padding-bottom: 5px;
+  padding-bottom: ${space(0.5)}px;
 `
 
 export const StyledInput = styled.input`
   ${borderedInput};
   ${block(24)};
 `
 
-const InputComponent = styled.input.attrs<{ showLabel: boolean }>({})`
-  ${garamond("s17")};
-  border: 0;
-  font-size: 17px;
-  outline: none;
-  flex: 1;
-  transition: all 0.25s;
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  padding: 0 ${space(1)}px;
-  line-height: initial;
-  ${props => props.showLabel && "padding: 10px 10px 0 10px"};
-`
-
-const InputContainer = styled.div.attrs<{
-  hasLabel?: boolean
-  hasError: boolean
-}>({})`
-  ${borderedInput};
-  margin-right: 0;
-  margin-top: 5px;
-  margin-bottom: 10px;
-  display: flex;
-  position: relative;
-  height: ${p => (p.hasLabel ? "40px" : "20px")};
-  flex-direction: row;
-  align-items: center;
-  box-sizing: content-box;
-`
-
-const Label = styled.label.attrs<{ out: boolean }>({})`
-  ${unica("s12", "medium")};
-  position: absolute;
-  left: 10px;
-  top: 7px;
-  visibility: ${p => (p.out ? "hidden" : "visible")};
-  animation: ${p => (p.out ? fadeOut : fadeIn)} 0.2s linear;
-  transition: visibility 0.2s linear;
-  z-index: 1;
-`
-
 export const Title = styled.div`
   ${garamond("s17")};
 `
 
 const Description = styled.div`
   ${garamond("s15")};
-  color: ${Colors.graySemibold};
-  margin: 3px 0 0;
+  color: ${color("black60")};
+  margin: ${space(0.3)}px 0 0;
 `
```

Comment:
Agree this is for a second PR, but wonder what the impact would be if we just went ahead with the new typography and worried about spot-checking issues with 1-2px differences after the fact. I believe everything before the new design system can be considered legacy and thus replaced: 

```jsx
<Description size="3" m={0.3}>
 ...
</Description>
```